### ISSUE TITLE
DEVENV Config setting

### ISSUE THREAD

**Issue by LlewellynAtBluestone** [almbox/vsts-ssis-extension#28]
I used VS2017 to create my package.  However, Hosted V2017 does not appear in my Devenv selection.  What would I need to do to get this selection o include VS2017.  VS2017 is installed on the build machine.  Its the same machine used to create the SSIS package in the first place.

**Comment by VincentVerweij** [comment]
I am having the same problem as mentioned above.

As stated on the marketplace page, I installed SSDT for VS2017.
I don't know whether the dropdown for Devenv Version is filled dynamically but for me it is only giving 12 and 14 while on the marketplace page under "How to setup Build Task", I see VS2017 in the screenshot.

Also at the bottom of the marketplace page, it is stated under Latest Updates 1.x: "Implemented option for Custom path to Devenv", but where are we supposed to provide that custom path as the Devenv Version dropdown doesn't allow any custom entries.

**Comment by LlewellynAtBluestone** [comment]
I ended up using a powershell script instead. The script has one line to run the build.

&"${env:ProgramFiles(x86)}\Microsoft Visual Studio\2017\Professional\Common7\IDE\devenv.com" "C:\Users\me\Downloads\vsts-agent-win-x64-2.140.0\_work\2\s\POC_SSIS.sln" /rebuild

I place the code in a .ps1 file and put it into the root of my repo.  I then used the Powershell config to reference the file and it worked.   I did a similar thing for the deployment where I used : -

ISDeploymentWizard /S /SP:C:\Users\me\Downloads\vsts-agent-win-x64-2.140.0\_work\2\s\POC_SSIS\bin\Development\POC_SSIS.ispac /DS:serverName /DP:/SSISDB/POC_SSIS/POC_SSIS

**Comment by VincentVerweij** [comment]
Yes that's a solution too, what I did is install SSDT 2013 and switched the dropdown to 12. For me it worked because my project was a SQL Server 2012 SSIS project.
